src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -1321,6 +1321,17 @@ def test_asan_double_free(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_libfuzzer_deadly_signal(self):
+    data = self._read_test_data('libfuzzer_deadly_signal.txt')
+    expected_type = 'deadly signal'
+    expected_state = 'NULL'
```

Comment:
This is wrong, it shouldn't be null. We should get a fuzzer name there. @oliverchang wdyt ?

Reply:
We should be parsing the stack here. Looks like the format in the test file is not something we recognise.

Reply:
This is fine as-is, see reason above.